libraries/botbuilder/tests/botFrameworkAdapter.test.js

```diff
@@ -1049,7 +1049,16 @@ describe('BotFrameworkAdapter', () => {
         });
 
         it(`should createConversation() with parameters.`, async () => {
-            await createConversation(reference, () => null, { createConversationParams: { isGroup: true } });
+            await createConversation(
```

Comment:
@stevengum looks like there was already a test for creating a conversation with parameters. I added the assertions below to ensure `isGroup` is propagated.